### ISSUE TITLE
framerate decreases drastically when many individuals moving around

### ISSUE THREAD

**Issue by fguzm** [Return-To-The-Roots/s25client#205]
many deer, slow framerate
many men, slow framerate
the more the slower, not really playable then

debian amd64

**Comment by Spikeone** [comment]
Have you installed the latest graphics driver? What's the frame rate when the game starts? 
Gpu,CPU,ram?

**Comment by fguzm** [comment]
Yes, latest *free* graphics driver (closed source not working with newer kernels, but maybe I should test it with an old kernel and original amd driver from the debian repository).
It is a new issue for me, newer had such speed problems with rttr, know this only from dosbox with real settlers2.

**Comment by Spikeone** [comment]
Well, I only know the windows perspective, but that often happened when using a graphics driver which seems not to support OpenGL well.
For example my NetBook (1gb ram, 1,33GHz dual i3) also starts at 30 Fps and drops fast to 15 - but that's not a recommended system. But I guess you got norm gaming specs.

**Comment by Flow86** [comment]
you have to use proprietary graphic driver since the free driver then seems not to have full hardware acceleration implmeneted :-/

**Comment by MarcusSt** [comment]
I would not exactly call this a bug of RttR. RttR uses OpenGL for rendering and especially for slow drivers that do hardware emulation, about nothing runs in parallel. That way, many primitives on screen can increment the rendering time dramatically.

**Comment by fguzm** [comment]
Vertex buffer objects was disabled.
After enabling it in the rttr settings all runs smooth again.
Thank you for mentioning OpenGL.
